apps/artist/routes.coffee

```diff
@@ -72,8 +73,22 @@ sd = require('sharify').data
   if req.user?
     req.user.fetch
       success: (model, resp, options) ->
-        res.locals.sd.CURRENT_USER = _.extend(resp, res.locals.sd.CURRENT_USER)
-        res.locals.sd.IS_PAYOFF = true
-        res.render 'payoff'
+        user = _.extend(resp, res.locals.sd.CURRENT_USER)
+        send =
+          query: payoffQuery,
+          variables:
+            artist_id: req.params.id
+          req: user: user
+
+        return if metaphysics.debug req, res, send
+        metaphysics send
+          .then ({ me }) ->
+            res.locals.sd.CURRENT_USER = user
+            res.locals.sd.INITIAL_ARTISTS = me.suggested_artists
+            res.locals.sd.IS_PAYOFF = true
+            res.render 'payoff',
+              name: user.name
+              href: "/artist/#{req.params.id}/follow"
```

Comment:
Clicking 'Done' will take you to this href, which will follow the artist and bring you back to the artist page.